code/modules/antagonists/disease/disease_mob.dm

```diff
@@ -138,6 +138,8 @@ the new instance inside the host to be updated to the template's stats.
 	. = ..()
 	if(!mind.has_antag_datum(/datum/antagonist/disease))
 		mind.add_antag_datum(/datum/antagonist/disease)
+	var/datum/atom_hud/medsensor = GLOB.huds[DATA_HUD_MEDICAL_ADVANCED]
```

Comment:
yep this code looks like it works

Reply:
me bad coder but it works (?)

Reply:
this is how huds work, it's dumb but yes it's not a mob proc it's a proc you call on the hud